### ISSUE TITLE
[IDEA]

### ISSUE THREAD

**Issue by TheAlienDrew** [ruffle-rs/ruffle#790]
Ability to set a resolution and record as video... that would be really helpful.

**Comment by manokara** [comment]
You can use [the exporter](https://github.com/ruffle-rs/ruffle#exporter) for that, and the `--width` and `--height` arguments to set a resolution. It currently just exports an image sequence though, so you'd need to convert them to a video using `ffmpeg` or another tool.

**Comment by TheAlienDrew** [comment]
True, but it would still be nicer to skip the middle man

**Comment by manokara** [comment]
Indeed, but I think this has really low priority for the devs, they're still deciding how to handle video decoding, what dependencies to use and such.

Could you change the issue title to something more descriptive? Like "Export frames to video", so they can keep track of it.

**Comment by TheAlienDrew** [comment]
Sure yeah I could do that, not sure why it got cut in the first place to just "[IDEA]"

**Comment by gsuberland** [comment]
I'm gonna +1 this idea. The frame export approach loses audio so it's not really a solution. Yeah you can play the thing back and screen capture it with something like OBS, but that's cumbersome and can't be done headless. Direct flash to video transcoding would be great.

**Comment by parkerlreed** [comment]
If you want to record the raw surface plus audio and are using Linux there's https://github.com/nowrep/obs-vkcapture
